application/templates/cms/edit_dimension.html

```diff
@@ -61,7 +61,7 @@ <h2 class="govuk-heading-m">Charts & Tables</h2>
                             {% if dimension.dimension_table and dimension.dimension_table.table_object is not none and dimension.dimension_table.table_object != '""' %}
                                 <tr class="govuk-table__row">
                                     <td class="govuk-table__header" scope="row">Table</td>
-                                    <td class="govuk-table__cell">{{ dimension.dimension_table.table_object.header }}</td>
+                                    <td class="govuk-table__cell">{{ dimension.dimension_table.title }}</td>
```

Comment:
I always thought that `table_object.header` was the first (header) row of the table with all the column headings.  But no, of course it wasn't.

Reply:
🙈